src/components/QueryEditor.js

```diff
@@ -178,14 +178,7 @@ export class QueryEditor extends React.Component {
   }
 
   _onKeyUp = (cm, event) => {
-    const code = event.keyCode;
-    if (
-      (code >= 65 && code <= 90) || // letters
-      (!event.shiftKey && code >= 48 && code <= 57) || // numbers
-      (event.shiftKey && code === 189) || // underscore
-      (event.shiftKey && code === 50) || // @
-      (event.shiftKey && code === 57) // (
-    ) {
+    if (event.key.length === 1 && event.key.match(/[a-zA-Z0-9_@(]/)) {
```

Comment:
define the regex outside of this function, so it's init'd once, then use `test` here instead of `match` for better perf. Use `^[...]$` will also only pass if the key is length == 1

```js
const AUTO_COMPLETE_AFTER_KEY = /^[a-zA-Z0-9_@(]$/;
```

Then

```js
if (AUTO_COMPLETE_AFTER_KEY.test(event.key)) {
```